addons/binding/org.openhab.binding.onkyo/ESH-INF/config/config.xml

```diff
@@ -29,10 +30,16 @@
 		</parameter>
 		<parameter name="volumeLimit" type="integer" min="0" max="100">
 			<label>Volume Limit</label>
-			<description>Limit maximum volume level to defined value.</description>
+			<description>Limit maximum volume level to defined percentage.</description>
 			<default>100</default>
 			<advanced>true</advanced>
 		</parameter>
+		<parameter name="volumeScale" type="decimal">
+			<label>Volume Scale</label>
+			<description>Applies a scale to the volume.</description>
+			<default>1</default>
```

Comment:
What about adding this:
```suggestion
			<options>
				<option value="1">0-100</option>
				<option value="2">0-100 in 0.5 steps</option>
				<option value="0.8">0-80</option>
				<option value="0.5">0-50</option>
			</options>
			<default>1</default>
```

Reply:
Neat, didn't know that was possible. Added this 👍

Reply:
Nice. With this you will have a dropdown in some UIs (e.g. Paper UI) to select one of the four possible values.